### PR TITLE
feat(pci): public cloud load balancer

### PR DESCRIPTION
**Pull request by radireddy**
Implement load balancer list and onboarding page for public cloud. Added new module for load balancer home and onboarding page.
#MANAGER-4398

### CODE REVIEW

File: packages/manager/modules/pci/src/projects/project/load-balancer/translations/Messages_fr_FR.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "pci_projects_project_load_balancer_list_title": "Load Balancer",
+  "pci_projects_project_load_balancer_list_name": "Nom",
+  "pci_projects_project_load_balancer_list_status": "Statut",
+  "pci_projects_project_load_balancer_list_ipAddres": "Adresse IP",
+  "pci_projects_project_load_balancer_list_healthy": "En bonne santé",
+  "pci_projects_project_load_balancer_list_last_applied_configuration": "Dernière configuration appliquée",
+  "pci_projects_project_load_balancer_list_reqs": "Requis / s",
```
Comment (FredericEspiau) [inline packages/manager/modules/pci/src/projects/project/load-balancer/translations/Messages_fr_FR.json]: What is that supposed to mean ?

File: packages/manager/modules/pci/src/projects/project/load-balancer/load-balancer.routing.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
+import map from 'lodash/map';
+
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state('pci.projects.project.loadbalancer', {
+    url: '/load-balancer',
+    component: 'ovhManagerPciProjectLoadBalancer',
```
Comment (FredericEspiau) [inline packages/manager/modules/pci/src/projects/project/load-balancer/load-balancer.routing.js]:
```suggestion
    component: 'pciProjectLoadBalancer',
```

File: packages/manager/modules/pci/src/projects/project/load-balancer/translations/Messages_fr_FR.json (hunk 1 above)
Comment (radireddy) [reply packages/manager/modules/pci/src/projects/project/load-balancer/translations/Messages_fr_FR.json]: in mockup it ays Reqs/s. Maybe it means a request

Comment (radireddy) [reply packages/manager/modules/pci/src/projects/project/load-balancer/translations/Messages_fr_FR.json]: it means requests per seconds

File: packages/manager/modules/pci/src/projects/project/load-balancer/load-balancer.controller.js [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+  getLoadbalancerDetails({ id: loadBalancerId }) {
+    return this.PciLoadBalancerService.getLoadBalancerDetails(
+      this.projectId,
+      loadBalancerId,
+    );
+  }
+
+  trackGuideClick(guide) {
+    if (guide.id === USING_LOAD_BALANCER_GUIDE_ID) {
+      this.PciLoadBalancerService.trackClick(LOAD_BALANCER_CONFIGURE_TRACKING);
```
[the remark this answers is not in the document]
Comment (radireddy) [reply packages/manager/modules/pci/src/projects/project/load-balancer/load-balancer.controller.js]: Not all guide items require tracking. Also, atInternet service is required for tracking. Adding this dependency on a reusable component requires other dependencies. I tried it and it threw a dependency error. I don't remember the exact dependencies.

**Comment by ovh-cds** [comment]
CDS Report build-monorepository#10304.0 ✘  
* 
  * Build ✘

**Comment by ovh-cds** [comment]
CDS Report build-monorepository#10379.0 ✘  
* 
  * Build ✘

**Comment by ovh-cds** [comment]
CDS Report translator-push#6629.0 ✘  
* 
  * push ✘

**Comment by ovh-cds** [comment]
CDS Report translator-push#6629.0 ✘  
* 
  * push ✘

**Comment by ovh-cds** [comment]
CDS Report translator-push#6631.0 ✘  
* 
  * push ✘

**Comment by ovh-cds** [comment]
CDS Report translator-push#6631.0 ✘  
* 
  * push ✘

**Pull request merged**